DSCResources/MSFT_xPowerPlan/MSFT_xPowerPlan.psm1

```diff
@@ -0,0 +1,108 @@
+function Get-TargetResource
+{
+    [CmdletBinding()]
+    [OutputType([System.Collections.Hashtable])]
+    param
+    (
+        [Parameter(Mandatory = $true)]
+        [ValidateSet('Yes')]
+        [System.String]
+        $IsSingleInstance,
```

Comment:
Hi @eshaparmar - this is the best practice for single instance resources. See this [MSDN page](https://msdn.microsoft.com/en-us/powershell/dsc/singleinstance). @TravisEz13 also wrote [this article](https://blogs.msdn.microsoft.com/powershell/2015/06/22/want-to-write-a-dsc-resource-where-only-a-single-instance-can-be-configured/) on it. :smile: